server/utils/validation/gymEditSchema.ts

```diff
@@ -0,0 +1,22 @@
+import Joi from 'joi';
+
+const gymRegisterSchema = Joi.object({
+  gymName: Joi.string().required(),
+  logo: Joi.string().required(),
+  email: Joi.string()
+    .max(250)
+    .min(4)
+    .email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } })
+    .required(),
```

Comment:
we will not send email on edit profile, nor password, an we will send gym ID in the request body